ballerina/tests/complex-query-test.bal

```diff
@@ -353,6 +463,33 @@ function testDateTime() returns error? {
     test:assertEquals(value, expected, "Expected record did not match.");
 }
 
+@test:Config {
+    groups: ["query", "query-complex-params"]
+}
+function testDateTimeRecord() returns error? {
+    MockClient dbClient = check new (url = complexQueryDb, user = user, password = password);
+    ResultDates value = check dbClient->queryRow("SELECT date_type, time_type, timestamp_type, datetime_type"
+       + ", time_tz_type, timestamp_tz_type from DateTimeTypes where row_id = 1", ResultDates);
```

Comment:
```suggestion
       + ", time_tz_type, timestamp_tz_type from DateTimeTypes where row_id = 1");
```

Reply:
Shall we avoid passing returnType to the call, It should infer it from the assigned variable type

Reply:
Fixed in 91e3f85